packages/gtmcore/gtmcore/inventory/inventory.py

```diff
@@ -445,7 +447,14 @@ def delete_labbook(self, username: str, owner: str, labbook_name: str) -> List[D
         SecretStore(lb, username).clear_files()
 
         # Remove labbook contents
-        shutil.rmtree(lb.root_dir, ignore_errors=True)
+        if 'LOCAL_USER_ID' not in os.environ:
```

Comment:
We also have "WINDOWS_HOST" (not defined except on Windows) which would be more obvious - but I guess perhaps we are expecting this might happen in other situations?
```suggestion
        if 'WINDOWS_HOST' in os.environ:
```